src/main/java/uk/gov/hmcts/reform/cpo/data/CasePaymentOrderEntity.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.cpo.data;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+
+import javax.persistence.Column;
+import javax.persistence.Id;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "case_payment_orders")
+public class CasePaymentOrderEntity {
+
+    @Id
+    @GeneratedValue
+    private UUID id;
+
+    @CreationTimestamp
+    private LocalDateTime createdTimestamp;
```

Comment:
mmm `LocalDateTime`? 🤔 on the conf page it says these are UTC times "YYYY-MM-DDTHH:MI:SSZ". Wonder whether there could be some issues with storing them as local dates

Reply:
https://i.stack.imgur.com/WOJAF.png

Reply:
To be resolved in separate PR as per discussion

Reply:
you can enforce that in the setter(..)  LocalDateTime.now(Clock.systemUTC()); 
So, obviously we should remove @CreationTimestamp